javascript-data-model/order-history.js

```diff
@@ -0,0 +1,106 @@
+var orderHistory = [
+  {
+    orderDetails: {
+      orderPlaced: 'Aug 04, 2020',
+      orderNumber: '114-3941689-8772232',
+      recipient: 'JS Masher'
+    },
+
+    itemDetails: {
+      itemName: ['JavaScript for impatient programmers'],
+      author: ['Rauschmayer, Dr.Axel']
+    },
+
+    deliveryDetails: {
+      delivered: true,
+      deliveryEstimate: 'Aug 08, 2020',
+      deliveryHandOff: false,
+      returnWindow: ['Sep 7, 2020'],
+      productSupport: false
+
+    },
+
+    costDetails: {
+      itemCost: [31.55],
+      cartTotal: 34
+    }
+  },
```

Comment:
I can see your thought process here, but this may be overly sub-divided. Also, you have a bunch of individual arrays for pieces of data for the items ordered, you should make it one array holding an object for each item with all the data for that item in that one object.

Reply:
Thanks for the feedback, I'll revise and get it back to you!